webpack.config.babel.js

```diff
@@ -1,18 +1,18 @@
-const path = require('path');
+const Path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
-const VisualizerPlugin = require('webpack-visualizer-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
 
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 2900000;
-const MAX_PRODUCTION_OUTPUT_SIZE = 335000;
+const MAX_DEVELOPMENT_OUTPUT_SIZE = 3300000;
+const MAX_PRODUCTION_OUTPUT_SIZE = 400000;
```

Comment:
I lowered the limit for production size a bit, but not that much.

Honestly I do not know what caused the difference. I did not make any significant changes in the actual code, so the increase must be due to the build process.

There were two changes:
1. migration to webpack5
2. building CSS and JS code as two separate files

Both files look OK. I have no idea what else to do with it :(

Reply:
In that case, leave it. But I think we will need to create optimization issue in the future. 400kB si to large for library like this, it's just few components. It might contains some code we don't need